src/bb_inits_and_defaults.jl

```diff
@@ -196,10 +192,6 @@ function init_juniper_problem!(jp::JuniperProblem, model::MOI.AbstractOptimizer)
             int_i += 1
         end
     end
-    jp.num_l_constr = num_linear_le_constraints+num_linear_ge_constraints+num_linear_eq_constraints
```

Comment:
is this information lost with the new change?

Reply:
Yes, it's not so well defined for a generic MOI model. For instance, do you count NormOneConstraint as linear ? As how many ? If you have a constraint in `MOI.Zeros(1000)` is it one constraint or `1000` ?
As it's only used in printing, we could add this back later with a more general printing that says, e.g. how many constraint there is of each type.